features/partners.feature

```diff
@@ -105,17 +107,17 @@ Functionaliteit: Huwelijken en geregistreerd partnerschappen van een persoon raa
       | adellijkeTitelPredicaat |         |
       | voorvoegsel             |         |
       | geslachtsnaam           | Groenen |
-      En heeft de partner met burgerservicenummer '555550008' de volgende 'geboorte' gegevens
+      En heeft de partner met burgerservicenummer '555550007' de volgende 'geboorte' gegevens
       | naam   | waarde   |
       | datum  | 19861004 |
       | plaats | 0503     |
       | land   | 6030     |
-      En heeft de partner met burgerservicenummer '555550008' de volgende 'aangaanHuwelijkPartnerschap' gegevens
+      En heeft de partner met burgerservicenummer '555550007' de volgende 'aangaanHuwelijkPartnerschap' gegevens
       | naam   | waarde   |
       | datum  | 20210720 |
       | plaats | Dunvegan |
       | land   | 6039     |
-      En heeft de partner met burgerservicenummer '555550008' de volgende 'ontbindingHuwelijkPartnerschap' gegevens
+      En heeft de partner met burgerservicenummer '555550007' de volgende 'ontbindingHuwelijkPartnerschap' gegevens
       | naam  | waarde   |
       | datum | 20220301 |
       En heeft de persoon met burgerservicenummer '555550003' géén partner met burgerservicenummer '555550008'
```

Comment:
dit scenario gaat over de persoon met bsn '555550006'

Reply:
Regel 123 komt niet in mijn scenario voor. Merge-issues